specs/LoaderDllFolderPath.md

```diff
@@ -0,0 +1,39 @@
+# Background
+For some reasons, there are some developers want to specify the path of 'WebView2Loader.dll' explicitly.
```

Comment:
```suggestion
The WebView2 loader code is what knows where to find and start or connect to already running WebView2 runtimes. This can either be statically linked for C based projects, or is also available as a standalone DLL. For .NET projects they must use the DLL since the loader code is native and we don't have a way to merge the native code into a managed module. Before this change the .NET WebView2 API module would look for the WebView2Loader.dll in the its same folder. But some .NET projects have requirements about where they can place DLLs and so this property is introduced to allow end developers to place the WebView2Loader.dll in any folder and specify the path of 'WebView2Loader.dll' explicitly.
```